distributed_shampoo/utils/shampoo_preconditioner_list.py

```diff
@@ -920,13 +922,30 @@ def _create_kronecker_factors_state_for_block(
             ),
         )
 
+    def _get_inverse_exponent_override_based(self, dimension: int, order: int) -> float:
```

Comment:
Why not just `_get_inverse_exponent_override`?

Reply:
I see, I actually think just `_get_inverse_exponent` would be a good name, the "override" doesn't add anything in this context.